uphold-integration/src/main/java/org/dash/wallet/integration/uphold/ui/UpholdAccountActivity.java

```diff
@@ -178,4 +194,81 @@ public void onTransfer() {
                 });
     }
 
+    private void revokeAccessToken() {
+        if (true) {
+            UpholdClient.getInstance().revokeAccessToken(new UpholdClient.Callback<String>() {
+                @Override
+                public void onSuccess(String result) {
+                    openUpholdToLogout();
+                }
+
+                @Override
+                public void onError(Exception e, boolean otpRequired) {
+                    showErrorAlert();
+                }
+            });
+        } else {
+            showAutoLogoutAlert();
+            openUpholdToLogout();
+        }
```

Comment:
These two lines will return the app to the UpholdSplash screen and show the uphold page so the user can log out manually.  We may need to put more instructions in the logout alert.